lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -381,15 +382,22 @@ namespace Js
 
 
         JavascriptSymbol* symbolMatch;
+        JavascriptSymbol* symbolReplace;
         JavascriptSymbol* symbolSearch;
         JavascriptSymbol* symbolSplit;
 
         UnifiedRegex::RegexPattern * emptyRegexPattern;
+        int regexConstructorSlotIndex;
         JavascriptFunction* regexExecFunction;
+        int regexExecSlotIndex;
         JavascriptFunction* regexFlagsGetterFunction;
+        int regexFlagsGetterSlotIndex;
         JavascriptFunction* regexGlobalGetterFunction;
+        int regexGlobalGetterSlotIndex;
         JavascriptFunction* regexStickyGetterFunction;
+        int regexStickyGetterSlotIndex;
         JavascriptFunction* regexUnicodeGetterFunction;
+        int regexUnicodeGetterSlotIndex;
```

Comment:
I would prefer if these two groups of fields were not interleaved in their pairs, but a pair of groups.  I think that will be faster to read.